egs2/libritts/speechlm1/conf/train.yaml

```diff
@@ -0,0 +1,51 @@
+# Model
```

Comment:
maybe consider only one validated config for the start.

Reply:
Sure. After we are sure the config is healthy. I'm trying multiple configs / archs since there could be a issue in current implementation